ee/kernel/src/initsys.c

```diff
@@ -14,10 +14,12 @@
 
 #include "kernel.h"
 #include "string.h"
+#include <timer_alarm.h>
 
 #ifdef F__InitSys
 void _InitSys(void)
 {
+    InitTimerAlarm();
```

Comment:
I think that following the other implementations it should be inside of the `#ifndef KERNEL_NO_PATCHES` what do you think @rickgaiser ?

Reply:
IIUC the KERNEL_NO_PATCHES define protects some functions that "patch" the BIOS (it literally writes over the BIOS memory patching some bugs in known code).
However I'm not doing any sort of patching.
Perhaps is not InitSys the right place to initialize an SDK lib? (In any case the ifdef doesnt apply here I think).

Reply:
Ok, got it, I think it is the proper place where it is then

Reply:
It should be indeed inside `#ifndef KERNEL_NO_PATCHES`, for those wanting to reduce code size (e.g. a memory card boot program).  
And if the user wants to initialize it manually using `libkernel-nopatch`, then they can do so, and the code will get linked. Otherwise it won't get linked and will not increase the code size.  
When using `libkernel`, it will be linked always.